deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/__init__.py

```diff
@@ -130,6 +140,24 @@ def insert_vulnerabilities_agent_inventory(agent_id='000', status='VALID'):
                                                          cve=package['cveid'], status=status)
 
 
+def insert_suse_system_package(agent_id='000', version='SLES15'):
+    """Insert suse OS package to an agent.
+
+    Args:
+        agent_id (str): Agent ID.
+        version (str): Package version.
+
+    Raises:
+        ValueError: If version parameter has an invalid value.
+    """
+    if version not in SUSE_SYSTEM_PACKAGE:
+        raise ValueError(f"Suse system parameter invalid.")
```

Comment:
`fstring` and `"` ?

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/2847/commits/349274a11fb90a6a8421a443306f248ef179d68e